docs/docs/download/index.md

```diff
@@ -0,0 +1,15 @@
+---
+authors: Fragcolor & contributors
+license: CC-BY-SA-4.0
+---
+
+# Download
+
+<figure markdown>
+  ![](assets/DownloadLogo.png)
+</figure>
+
+Download [here](https://github.com/fragcolor-xyz/shards/releases).
+
```

Comment:
Should we add a link to the primer here? Something like:

Unsure of where to go from here? Check out our [primers](../link) to learn more, or follow a [step-by-step tutorial](../link) to create a minigame with Shards!

Reply:
Good point. There are opportunities to link pages together a bit more.

If we keep the contribute pages, we could add a link to "building shards from source" as well.